superset/examples/big_data.py

```diff
@@ -68,3 +70,7 @@ def load_big_data() -> None:
     ]
     for i in range(1000):
         add_data(columns=columns, num_rows=10, table_name=f"small_table_{i}")
+
+    print("Creating table with long name")
+    name = "".join(random.choices(string.ascii_letters + string.digits, k=64))
```

Comment:
I think @amitmiran137 already create a `get_random_string` function in test utils: 
https://github.com/apache/superset/blob/9a7fba810e5bc3a94746117937163c1bfaea19ee/tests/dashboards/dashboard_test_utils.py#L99

Maybe  consolidate?

Reply:
I don't think it's possible to import the `get_random_string` function here, so are you suggesting moving it outside the `tests/` directory and reusing it both here and in the test?